deploy/redis/values.yaml

```diff
@@ -13,6 +13,20 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: 7.0.6-RC8
 
+redisImage:
+  registry: docker.io
+  repository: malexer/twemproxy
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
```

Comment:
Avoid using the 'latest' image tag. Please specify a specific version for the image tag.